File: PHP/Calculator/src/CalculatorBundle/Entity/Calculator.php
<?php
namespace CalculatorBundle\Entity;

class Calculator
{
    /**
     * @var float
     */
    private $leftOperand;

    /**
     * @var float
     */
    private $rightOperand;

    /**
     * @var string
     */
    private $operator;

    /**
     * Get left operand
     *
     *
     */
    public function getLeftOperand()
    {
        return $this->leftOperand;
    }

    /**
     * Set left operand
     *
     * @param float $leftOperand
     *
     */
    public function setLeftOperand(float $leftOperand)
    {
        $this->leftOperand = $leftOperand;
    }

    /**
     * Get right operand
     *
     */
    public function getRightOperand()
    {
        return $this->rightOperand;
    }

    /**
     * Set right operand
     *
     * @param float $rightOperand
     *     *
     */
    public function setRightOperand(float $rightOperand)
    {
        $this->rightOperand = $rightOperand;
    }

    /**
     * Get operator
     *
     */
    public function getOperator()
    {
        return $this->operator;
    }

    /**
     * Set operator
     *
     * @param string $operator
     *
     */
    public function setOperator( string $operator)
    {
        $this->operator = $operator;
    }

}

File: Java/Calculator/src/main/java/com/softuni/entity/Calculator.java
package com.softuni.entity;

import java.text.DecimalFormat;

public class Calculator {
    private double leftOperand;
    private double rightOperand;
    private String operator;

    public Calculator() {
    }

    public Calculator(double leftOperand, double rightOperand, String operator) {
        this.leftOperand = leftOperand;
        this.rightOperand = rightOperand;
        this.operator = operator;
    }

    public double getLeftOperand() {
        return leftOperand;
    }

    public void setLeftOperand(double leftOperand) {
        this.leftOperand = leftOperand;
    }

    public double getRightOperand() {
        return rightOperand;
    }

    public void setRightOperand(double rightOperand) {
        this.rightOperand = rightOperand;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public double calculateResult(){
        double result=0.0;
        if (this.operator.equals("+")){
            return this.leftOperand+this.rightOperand;
        }
        else if (this.operator.equals("-")){
            return this.leftOperand-this.rightOperand;
        }
        else if (this.operator.equals("/")){
            return this.leftOperand/this.rightOperand;
        }
        else if (this.operator.equals("*")){
            return this.leftOperand*this.rightOperand;
        }
        return result;
    }
}
